79 xml validation workflow

ryanmwhitephd opened the pull request:
Try to add xml validation workflow.

dwloup69 commented:
Same for the path of the MeasurandTaxonomyCatalog namespace setting.

ryanmwhitephd commented:
We discussed the namespace path during the last meeting. Can we keep the path as is? This will allow to publish a stable version elsewhere. We'll need to sort out the versioning of github releases and the schema, etc.. later.

Let us know if there is an issue with keeping the namespace path at https://cls-schemas.s3.us-west-1.amazonaws.com/MII/.

@SchwartzMichael

dwloup69 commented:
We'll need to discuss the casing of the names for MTC objects. Lower case timeinterval is hard to read where TimeInterval is easier to see. 
I talked with Mike and understand you guys discussed this on Monday. You can leave them as is.

ryanmwhitephd commented:
Casing of names for mtc objects depends xsd:Name datatype requirements: https://www.datypic.com/sc/xsd/t-xsd_Name.html
I'll look again, but underscores work TimeInterval -> time_interval

ryanmwhitephd commented:
@MarkKuster 
The following changes made as requested:
- Removed the erroneous reference to catalytic concentration.
- Removed any mLayer references related to UOM:Quantity ratio
- Added relative humidity mLayer references
- Change all mtc:mLayer aspect to use ml_name, e.g. as_acceleration, as_relative_humidity
- Angle parameters reference as_plane_angle
-  Fixed a few naming typos, e.g. Distaince, Lenght.

As well, this includes fixing the Parameter names to CamelCase.
If this addresses the changes, see if you can accept the changes and approve the merge.

dwloup69 commented:
I have pushed the updated xml and xsd files to AWS S3.
